app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -1,10 +1,30 @@
-p
+p.lead
   '#{@first_sentence} To confirm your email address, please click on the link
   'below, or copy and paste the entire link into your browser.
 
+table.button.expanded.large.radius
+  tbody
+    tr
+      td
+        table
+          tbody
+            tr
+              td
+                center
+                  = link_to 'Confirm this email', \
```

Comment:
When i read this it sounds like I'm being asked to confirm the 'email message' as opposed to the 'email address'.  That might just me though.